src/docs/releases/1.6.0.md

```diff
@@ -28,6 +28,10 @@ Here is a list of the new permissions
  | `DeleteUsersInRole_{0}` | allows deleting users in a given role. |
  | `AssignRoleToUsers_{0}` | allows assigning users to a given role. |
 
+ ### `OrchardCore.Search` Module
+
+The property name in the `SearchSettings` class was changed from `SearchProviderAreaName` to `ProviderName`. If you are using both `Lucene` and `Elasticsearch` provides at the same time, you'll need to update the search settings by setting the default search provides to one. 
```

Comment:
Not sure but if I understand correctly `provides` => `providers`
`the default search provides` => `the default search provider`